addons/dark-www/addon.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Dark WWW",
```

Comment:
It should probably be called *Website dark mode*, for consistency with *Editor dark mode* and because more variants will be added in the future.

Reply:
sure!

Reply:
done! although the folder name is still dark-www

Reply:
> done! although the folder name is still dark-www

That doesn't really matter.